students/17th/team1/hanahyang/user/views.py

```diff
@@ -0,0 +1,60 @@
+import json, re, traceback
+
+from django.views           import View
+from django.http            import JsonResponse
+from django.db.models       import Q
+from django.core.exceptions import ValidationError
+
+from .models                import User
+
+def validate_email(email):
+    pattern = re.compile('^.+@+.+\.+.+$')
+    if not pattern.match(email):
+        raise ValidationError('Invalid Email Format')
+
+def validate_password(password):
+    if len(password) < 8:
```

Comment:
```suggestion
    if len(password) < MINIMUM_PASSWORD_LENGTH:
```
8 같은 비교수치를 하드코딩보다는 상수화하는게 좋습니다.